gtsfm/common/two_view_estimation_report.py

```diff
@@ -34,6 +33,9 @@ class TwoViewEstimationReport:
         U_error_deg: relative translation error w.r.t. GT. Only defined if GT poses provided.
         i2Ri1: relative rotation.
         i2Ui1: relative translation direction.
+        reproj_error_gt_model: reprojection error betwen correspondences w.r.t. GT.
```

Comment:
thanks for adding these. small typo here on "between" I might say 
`reprojection errors between correspondences w.r.t. GT.` to indicate its plural

Reply:
fixed